contents/founders/write-great-job-ads.md

```diff
@@ -0,0 +1,92 @@
+---
+date: 2024-06-13
+title: Obvious and non-obvious ways to write great job ads
+author:
+  - charles-cook
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/transparency.jpeg
+featuredImageType: full
+tags:
+  - Culture
+  - People
+crosspost:
+  - Blog
+  - Newsletter      
+---
+
+At PostHog, we've written job ads that have seen 9,000 applicants in the last 12 months. One thing we've noticed in 2024 is that the quality of the average applicant has gone way, way up. We think this is due to a couple of things:
+
+- We've started putting a lot more effort into making our job ads more interesting and differentiated
+ 
+- We've stopped advertising anywhere except on our own website (of course we still pop up on a few scraper sites)
+
+Given the number of people I've interviewed recently who have said 'I loved your job ad', we figure some of this stuff must be working! Here is a super-quick tactical guide which will hopefully help. 
```

Comment:
```suggestion
Given the number of people we've interviewed recently who have said 'I loved your job ad', we figure some of this stuff must be working! Here is a super-quick tactical guide which will hopefully help you if you're advertising for a job, or applying for one. 
```

Reply:
Not going to include this one, as this is only designed to help you write a job ad (not apply for one)